suro-elasticsearch/src/main/java/com/netflix/suro/sink/elasticsearch/ElasticSearchSink.java

```diff
@@ -264,90 +160,199 @@ public String getStat() {
     @Override
     protected void beforePolling() throws IOException {}
 
-    private IndexRequest createIndexRequest(Message m) {
+    private void createClient() {
+        if (ribbonEtc.containsKey("eureka")) {
+            ribbonEtc.setProperty(
+                clientName + ".ribbon.AppName", clientName);
+            ribbonEtc.setProperty(
+                clientName + ".ribbon.NIWSServerListClassName",
+                "com.netflix.niws.loadbalancer.DiscoveryEnabledNIWSServerList");
+            ribbonEtc.setProperty(
+                clientName + ".ribbon.ServerListRefreshInterval",
```

Comment:
Why do you need to override the default refresh interval? Longer interval may lead to staled discovery data.